app/service/zendesk_ticket_service.rb

```diff
@@ -0,0 +1,44 @@
+class ZendeskTicketService
+  class << self
+    def create!(...)
+      new.create!(...)
+    end
+  end
+
+  def create!(comment:, **params)
+    ticket_json = { ticket: {
+      comment:,
+      **params,
+    } }.to_json
+
+    post_json!("https://#{subdomain}.zendesk.com/api/v2/tickets.json", ticket_json, headers:)
+  end
+
+private
```

Comment:
None of these methods look like anything that need to be private?

Reply:
What makes you say that? I'm asking out of ignorance 😅 I just thought that since I didn't expect any callers of the service object to use them, they could be private.

Reply:
I think I'm being overzealous with my "too many private methods" ray, but in this case it looks ok.